ci/run_travis_commands.sh

```diff
@@ -11,6 +11,7 @@ fi
 
 cmake -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE} \
       -DCMAKE_CXX_FLAGS="${FLAGS}" \
+      -DENABLE_LINE_EDITING="${ENABLE_LINE_EDITING}" \
```

Comment:
Isn't this doing the same as the `ENABLE_LINE_EDITING` two lines below?

Reply:
Yes, looks like that :)

I guess we won't need this one!